Templates/VIPER Module Unit Tests.xctemplate/___VARIABLE_moduleName___InteractorTests.swift

```diff
@@ -9,16 +9,18 @@
 import XCTest
 @testable import ___PROJECTNAME___
 
-class ___VARIABLE_moduleName___InteractorTests: XCTestCase {
+final class ___VARIABLE_moduleName___InteractorTests: XCTestCase {
+
     var interactor: ___VARIABLE_moduleName___Interactor!
-    var interactorOutput: ___VARIABLE_moduleName___InteractorOutputMock!
```

Comment:
I think let's stick with `interactorOutput` instead of `presenter` as we only focus on Interactor in this test here. How about that? 🤔